src/Autoload.php

```diff
@@ -10,22 +10,8 @@
 
 Plugin::uses(InteractsWithLivewire::class);
 
-if (class_exists(Testable::class)) {
-    /**
-     * @return Testable // @phpstan-ignore-next-line
-     */
-    function livewire(string $name, array $params = [])
-    {
-        // @phpstan-ignore-next-line
-        return test()->livewire(...func_get_args());
-    }
-} else {
-    /**
-     * @return TestableLivewire
-     */
-    function livewire(string $name, array $params = [])
-    {
-        // @phpstan-ignore-next-line
-        return test()->livewire(...func_get_args());
-    }
-}
+function livewire(string $name, array $params = [])
```

Comment:
We can remove the 2 Livewire imports at the top then. 👍🏻 It's a shame that we won't have autocompletion, but 🤷🏻